spec/unit/configurer_spec.rb

```diff
@@ -213,7 +213,7 @@
       }
       Puppet::Node::Facts.indirection.save(facts)
 
-      expect(Puppet).to receive(:warning).with(/Fact value '.+' with the value length: '[1-9]*' exceeds the value length limit: [1-9]*/).twice
+      expect(Puppet).to receive(:warning).with(/Fact '.+' with value '.+' with the value length: '[1-9]*' exceeds the value length limit: [1-9]*/).twice
```

Comment:
Ah it looks like the joined fact name doesn't include the dot, so the actual message looks like:

```
Fact 'processorsisa' with value 'i386' with the value length: '4' exceeds the value length limit: 1
```

I think we want to report the dotted name like:

```
Fact 'processors.isa' with value 'i386' with the value length: '4' exceeds the value length limit: 1
```

Could you also update the `.+` matcher to be the literal string `processors.isa` so it's clear we're matching the dotted name?

Reply:
Ooo I didn't catch that! Since multiple facts are being tested from the `fact.values`, I wouldn't be able to match the literal string if we still wanted to test multiple facts. 

Instead I had it match the regex `.+\..+` which checks that there is a dot in the middle. If it would be better to just test one fact, like processors.isa, instead of several facts and have it match the literal string `processors.isa`, I can do that instead though!